apps/tester-federation/package.json

```diff
@@ -20,16 +20,19 @@
     "@react-native-async-storage/async-storage": "^1.23.1",
     "@react-navigation/native": "^6.1.18",
     "@react-navigation/native-stack": "^6.10.1",
-    "react": "18.2.0",
-    "react-native": "0.74.3",
-    "react-native-safe-area-context": "^4.10.8",
-    "react-native-screens": "^3.32.0"
+    "react": "18.3.1",
+    "react-native": "0.76.0-rc.1",
+    "react-native-safe-area-context": "^4.11.0",
+    "react-native-screens": "^4.0.0-beta.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.23.9",
-    "@react-native/babel-preset": "0.74.83",
-    "@react-native/eslint-config": "0.74.83",
-    "@react-native/typescript-config": "0.74.83",
+    "@babel/core": "^7.25.2",
+    "@react-native/babel-preset": "0.76.0-rc.1",
+    "@react-native/eslint-config": "0.76.0-rc.1",
+    "@react-native/typescript-config": "0.76.0-rc.1",
+    "@react-native-community/cli": "15.0.0-alpha.2",
+    "@react-native-community/cli-platform-android": "15.0.0-alpha.2",
+    "@react-native-community/cli-platform-ios": "15.0.0-alpha.2",
```

Comment:
do we need to add all these 3 packages explicitly? 

cc @szymonrybczak

Reply:
The RN upgrade helper specifies this explicitly, but now I see that RNTA already has them: https://github.com/microsoft/react-native-test-app/blob/a89b47711dd2ddd5a92d853267204af644a76d66/package.json#L171 so most likely we can skip this

Reply:
these are just resolutions, not dependencies though

Reply:
Removed in 2cdf84773fbe164e37ec0a53125d8b09d0b85b24